### ISSUE TITLE
spec clarification: allow multiple open() and close() requests, and multiple calls to requestMIDIAccess()

### ISSUE THREAD

**Issue by toyoshim** [WebAudio/web-midi-api#128]
For now the spec say nothing about multiple outstanding open() or close() requests.
But, what happens if open() is called before the previous open() is not resolved?

I feel it's ok to call async function that returns Promise many times before the previous call finishes.
Before JavaScript has the Promise, typical async APIs, e.g., File API, does not allow multiple requests, and the second request will throw InvalidStateError. But we can manage multiple requests with Promise.

**Comment by cwilso** [comment]
This should also apply to multiple requestMIDIAccess calls - where the app may decide to time out and re-request without sysex, for example.

**Comment by toyoshim** [comment]
Agreed. Current Chrome implementation also allows multiple requestMIDIAccess calls.

**Comment by agoode** [comment]
A question about Promises. Would the spec allow implementations to return the **same** Promise object for multiple open() calls?

**Comment by cwilso** [comment]
I think you actually want to have different promises.
